tests/phpunit/tests/kses.php

```diff
@@ -1353,6 +1354,21 @@ public function data_safecss_filter_attr() {
 				'css'      => 'background-repeat: no-repeat',
 				'expected' => 'background-repeat: no-repeat',
 			),
+			// Convert "&amp;" to "&".
+			array(
+				'css'      => 'background: url("/sites/2/2023/10/image.jpg?width=1024&amp;height=600")',
+				'expected' => 'background: url("/sites/2/2023/10/image.jpg?width=1024&height=600")',
+			),
+			// Continues to handle "&".
+			array(
+				'css'      => 'background: url("/sites/2/2023/10/image.jpg?width=1024&height=600")',
+				'expected' => 'background: url("/sites/2/2023/10/image.jpg?width=1024&height=600")',
+			),
+			// Background-image with valid entities including &amp;.
+			array(
+				'css'      => 'background-image:url("/sites/2/2023/10/some%20%21%40-_+-&image.jpg?foo=some%20%21%40-_+thing&width=1024&height=600")',
+				'expected' => 'background-image:url("/sites/2/2023/10/some%20%21%40-_+-&image.jpg?foo=some%20%21%40-_+thing&width=1024&height=600")',
+			),
```

Comment:
Added this dataset to make sure valid entities continue to be handled before and after this change ✅